lib/menubar/init.lua

```diff
@@ -53,6 +53,10 @@ end
 --- Menubar normal background color.
 -- @beautiful beautiful.menubar_bg_normal
 -- @param color
+--
```

Comment:
```suggestion

```

I think the doc generation error is because of this empty comment. The line should be blank.